homeassistant/components/media_player/epson.py

```diff
@@ -142,11 +142,13 @@ def supported_features(self):
         """Turn on epson."""
         from epson_projector.const import TURN_ON
         await self._projector.send_command(TURN_ON)
+        self.async_update_ha_state(force_refresh=True)
```

Comment:
This isn't necessary since this is a polling entity. At the end of the service call a state update will be scheduled.

Reply:
So what could be wrong that without that it is switching on (for about 30 seconds) -> off (30 seconds) -> on?

Reply:
Did you test all services?